src/pages/AdminTemplates/AdminTemplates.tsx

```diff
@@ -0,0 +1,154 @@
+import classNames from "classnames";
+import React, { useState } from "react";
+
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { Room } from "types/rooms";
+
+import { PartyMapContainer } from "pages/Account/Venue/VenueMapEdition";
+
+import "./AdminTemplates.scss";
+
+interface TemplateType {
+  title: string;
+  description: string;
+  background: string;
+}
+
+interface RecipeType {
+  title: string;
+  rooms: Room[];
+}
+
+const templates: TemplateType[] = [
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+  {
+    title: "Title",
+    description: "Description",
+    background: DEFAULT_MAP_BACKGROUND,
+  },
+];
+
+const recipes: RecipeType[] = [
+  { title: "Recipe 1", rooms: [] },
+  { title: "Recipe 2", rooms: [] },
+  { title: "Recipe 3", rooms: [] },
+  { title: "Blank Canvas", rooms: [] },
+];
+
+export const AdminTemplates: React.FC = () => {
```

Comment:
This is probably valid, and I expect we'll want to extract chunks of this component into sub-components rather than having it all defined in this one place.